sdk/storage/Azure.Storage.Internal.Avro/src/Record.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Azure.Storage.Internal.Avro
+{
+    internal class Record
```

Comment:
what is this for?

Reply:
Deleted, it was an artifact of a previous implementation attempt.